CHANGELOG-0.x.md

```diff
@@ -1,3 +1,38 @@
+# v0.8.0
+
+## Notable changes
+- Images will be built on a Debian base by default. Images built on Amazon Linux will still be available but with the tag suffix `-amazonlinux`.
```

Comment:
Let's definitely add `gp3 is now default` here.

Reply:
done